plugins/modules/query_graphql.py

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/python
 """Ansible plugin definition for query_graphql action plugin."""
-from __future__ import absolute_import, division, print_function
```

Comment:
I believe this is required by Ansible on all module definitions for the Ansible testing procedures. I'm good to remove if the tests pass without them as they definitely are not needed.

Reply:
The imports are in fact needed for the linting:

```
ERROR: The 1 sanity test(s) listed below (out of 32) failed. See error output above for details.
future-import-boilerplate```

Reply:
I think you can safely remove this test from the ansible sanity tests, it is intended for Python 2/3, but this repo only support Python 3. This would come into conflict with other linters for unused import.

![image](https://user-images.githubusercontent.com/9260483/124061623-38554e00-d9fd-11eb-8ab9-f6793b780da3.png)

Reply:
👍🏻